### DISCUSSION TITLE
Hybrid basemap not visible in 02_maps.ipynb

### DISCUSSION THREAD

**Discussion by Sumana18** [giswqs/geebook#4] [Q&A] [answered]
While running the code as it is on google colab , I noticed the Hybrid basemap is not appearing. Am I doing something wrong?

![image](https://github.com/giswqs/geebook/assets/40293452/7e1ca607-a38e-4693-a556-cf084a876fea)

**Comment by RoamingDutchman** [comment]
No it seems that map is not available. Others work fine.

**Reply by giswqs** [reply, marked as answer]
See https://github.com/gee-community/geemap/pull/1681#issuecomment-1707109740
